nuxeo-features/nuxeo-thumbnail/src/test/java/org/nuxeo/ecm/platform/thumbnail/test/TestRecomputeThumbnails.java

```diff
@@ -123,4 +123,20 @@ public void testRecomputeThumbnails() throws Exception {
         assertNotNull(thumbnail);
     }
 
+    @Test
+    public void testThumbnailsForTIFF() throws Exception {
+
+        DocumentModel doc = session.createDocumentModel("/", "testDoc", "File");
+        Blob blob = Blobs.createBlob(FileUtils.getResourceFileFromContext("test-data/at3_1m4_01.tif"), "image/tiff",
+                StandardCharsets.UTF_8.name(), "at3_1m4_01.tif");
+        doc.setPropertyValue("file:content", (Serializable) blob);
+        doc = session.createDocument(doc);
+
+        // wait for thumbnail generation
+        txFeature.nextTransaction();
+        doc = session.getDocument(doc.getRef());
+        Blob thumbnail = thumbnailService.getThumbnail(doc, session);
+        assertNotNull("the TIFF thumbnail is not generated", thumbnail);
```

Comment:
no need for the message `"the TIFF thumbnail is not generated"` just assertNotNull(thumbnail) is enough.

Reply:
log message are more explicit when the assertion fail and watching log in long jenkins log file!

Reply:
yes i know, but in this case the default message will be enough, and it's what we did in Nuxeo to be consistent with other cases.

some examples of assertions from the same class test: [here](https://github.com/nuxeo/nuxeo/blob/0d3268d8b73f4c3ad270c20dce187c863a53247a/nuxeo-features/nuxeo-thumbnail/src/test/java/org/nuxeo/ecm/platform/thumbnail/test/TestRecomputeThumbnails.java#L117) and [here](https://github.com/nuxeo/nuxeo/blob/0d3268d8b73f4c3ad270c20dce187c863a53247a/nuxeo-features/nuxeo-thumbnail/src/test/java/org/nuxeo/ecm/platform/thumbnail/test/TestRecomputeThumbnails.java#L105)